cpp/tests/rolling/grouped_rolling_test.cpp

```diff
@@ -2413,3 +2414,38 @@ TYPED_TEST(TypedUnboundedWindowTest, UnboundedPrecedingAndFollowingWindowMultiGr
                                  fixed_width_column_wrapper<cudf::size_type>{
                                    {3, 3, 3, 3, 3, 4, 4, 4, 4, 4}, {1, 1, 1, 1, 1, 1, 1, 1, 1, 1}});
 }
+
+TYPED_TEST(TypedUnboundedWindowTest, UnboundedPrecedingAndFollowingStructGroup)
+{
+  // Test that grouping on STRUCT keys produces is possible.
+
+  using namespace cudf;
+  using namespace cudf::test;
```

Comment:
Seems this function is both using these and not using these.
I recommend removing these and adding the namespace to the very few places here.
```suggestion
```

Reply:
Yikes, I'll clean that up.